### ISSUE TITLE
Could not find com.github.chrisbanes:PhotoView:1.3.1

### ISSUE THREAD

**Issue by JkMulti** [zixpo/candybar#207]
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform PhotoView-1.3.1.aar (com.github.chrisbanes:PhotoView:1.3.1) to match attributes {artifactType=android-aar-metadata, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Could not find PhotoView-1.3.1.aar (com.github.chrisbanes:PhotoView:1.3.1).
        Searched in the following locations:
            https://jitpack.io/com/github/chrisbanes/PhotoView/1.3.1/PhotoView-1.3.1.aar
            https://jitpack.io/com/github/chrisbanes/PhotoView/1.3.1/PhotoView-1.3.1.jar

**Comment by Donnnno** [comment]
I've fixed this in candybar-foss yesterday, and planning on making a PR here too! 
https://github.com/Donnnno/candybar-foss/commit/709b4aadc1d1d9c75c5c9b250800f1dfbf586aa8
https://github.com/Donnnno/candybar-foss/commit/1ce0a89d75c9174eab2cb262e072ba28f99338ec

**Comment by JkMulti** [comment]
I am still getting error. Do I need to update anything??

**Comment by Donnnno** [comment]
Gotta wait now till the next version of candybar gets released

**Comment by JkMulti** [comment]
When it's expected to release?

**Comment by sarsamurmu** [comment]
I am trying to update soon, but there's some problems occurring in my android studio. I will as soon as I get my android studio working.

**Comment by ladle3000** [comment]
Any idea when this will be up? This was my first attempt at making an apk for a few custom folder icons I want to install as an apk. I hit these same errors, and thought it was due to my noobness. 

Started working it out with gpt 4o which almost got me there:

![Screenshot_20241221-073649.png](https://github.com/user-attachments/assets/c3b34743-cebe-43ee-bdba-28a3e3e300d6)

![Screenshot_20241221-073702.png](https://github.com/user-attachments/assets/85e75288-8bee-4240-8f76-3b37e953d3ac)

But it turned into a rabbit hole whack a mole situation and I jumped ship. 

Long story short, the edge gestures extension I'm using doesn't allow custom icons from gallery, so I need to install as an icon pack. Making my own pack is something I've been wanting to do anyway. 

Is it normally pretty straight forward when these issues aren't present?

**Comment by Donnnno** [comment]
@ladle3000 please have some patience,
it's already implemented in https://github.com/zixpo/candybar/pull/208

You can use the snapshot version of candybar as a dependency meanwhile:

```
	dependencies {
	        implementation 'com.github.zixpo:candybar:-SNAPSHOT'
	}
```

**Comment by ladle3000** [comment]
> @ladle3000 please have some patience,
> it's already implemented in https://github.com/zixpo/candybar/pull/208
> 
> You can use the snapshot version of candybar as a dependency meanwhile:
> 
> ```
> 	dependencies {
> 	        implementation 'com.github.zixpo:candybar:-SNAPSHOT'
> 	}
> ```
> 

I used your foss version

**Comment by MiepHD** [comment]
> @ladle3000 please have some patience,
> it's already implemented in https://github.com/zixpo/candybar/pull/208
> 
> You can use the snapshot version of candybar as a dependency meanwhile:
> 
> ```
> 	dependencies {
> 	        implementation 'com.github.zixpo:candybar:-SNAPSHOT'
> 	}
> ```
> 

Using -SNAPSHOT didn't work for me. It said this could not be found.

**Comment by Donnnno** [comment]
@MiepHD please try with 3.21.0 it's the latest version!
It released yesterday

**Comment by MiepHD** [comment]
Thanks. That'll work

**Comment by ladle3000** [comment]
> I've fixed this in candybar-foss yesterday, and planning on making a PR here too! [Donnnno/candybar-foss@709b4aa](https://github.com/Donnnno/candybar-foss/commit/709b4aadc1d1d9c75c5c9b250800f1dfbf586aa8) [Donnnno/candybar-foss@1ce0a89](https://github.com/Donnnno/candybar-foss/commit/1ce0a89d75c9174eab2cb262e072ba28f99338ec)

What is the difference between this zixpo project and your own version candybar-foss?

**Comment by Donnnno** [comment]
@ladle3000 the FOSS version is compatible with the F-Droid store, removing all the non-open source stuff, like: license checker, billing and other google play stuff.

**Comment by ladle3000** [comment]
> @ladle3000 the FOSS version is compatible with the F-Droid store, removing all the non-open source stuff, like: license checker, billing and other google play stuff.

Ah. Thanks for the clarification.
